NetflixRouletteWebpack/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
@Marysosh Please add new line in end of file

Reply:
Corrected